src/test/components/ProfileCallTreeView.test.js

```diff
@@ -61,7 +61,7 @@ describe('calltree/ProfileCallTreeView', function() {
       profile = getProfileFromTextSamples(`
         A  A  A
         B  B  B
-        C  C  H[lib:libH.so]
+        C  C  H[lib:libH.so][file:hg:hg.mozilla.org/abdghij:a/random/path.rs:997a00815a6ba28806b75448a8829a0259d2ab28]
```

Comment:
Thanks for taking the time to improve the tests for this change!
Do you think you can add one for the s3 case, maybe for `I` ?

Reply:
Good idea.
To be honest, the biggest struggle in this test was the fact that I needed to avoid the letters "c", "f" and "e" because one part of this test filters by those letters and I didn't want to change which samples passed the filter.

Reply:
I ended up adding a new snapshot for this, with a new profile. Adding the file to `I` didn't work because it wasn't displayed in the call trees of the existing tests (the `H` node was collapsed by default).